test/activerecord_spanner_mock_server/spanner_active_record_with_mock_server_test.rb

```diff
@@ -1335,6 +1335,30 @@ def test_upsert_all_dml
       assert_equal 1, execute_requests.length
     end
 
+    def test_upsert_all_dml_with_exclude_from_change_streams
+      sql = "INSERT OR UPDATE INTO `singers` (`id`,`first_name`,`last_name`) " +
+            "VALUES (1, 'Dave', 'Allison'), (2, 'Alice', 'Davidson'), (3, 'Rene', 'Henderson')"
```

Comment:
nit: This SQL statement would in the real world not return 3 rows, it would only return an update count. It would slightly improve the readability of this test to add `THEN RETURN *` to the SQL string, so it matches what the mock return value is.